src/gettingstarted/cli/api-tokens.md

```diff
@@ -67,5 +69,4 @@ crons:
 ```
 
 > **note**
->
```

Comment:
Stylistically we have the extra line break in most places.  Don't mess with it here.  If we want to standardize that docs-wide that would be a separate PR task for just that.